recipe/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
   run:
     - zlib
     - popt 1.16
+    - libiconv
```

Comment:
@dfornika that is not the issue. The issue is that pinning to popt. popt depends on libiconv, but this old version of popt has not pinned libiconv ... so using a newer popt version would be the correct fix I think.

Reply:
popt has, as well as zlib, run_exports defined. So in theory you can remove the pinning from popt host dependency and remove all run-dependencies.

Reply:
Ok, thanks for that insight. I'll close this PR.

Reply:
Oh no please reopen ... I will try to modify it ...